ui/apps/ui/src/app/components/search-input/search-input.component.ts

```diff
@@ -281,7 +281,7 @@ export class SearchInputComponent implements OnInit {
       queryParams: {
         q: sanitizeQuery(q) ?? '*',
       },
-      //queryParamsHandling: 'merge',
+      queryParamsHandling: 'merge',
```

Comment:
I'm not 100% if this is behavior we desire. This might break some things.

Reply:
I consulted and this change was tested.